### PR TITLE
Always run anaconda private dbus session

### PR DESCRIPTION
**Pull request by jkonecny12**
We have our private DBus with our specific configuration.
Our private DBus is launched every time when Anaconda is started.
We are using our dbus variable instead of `DBUS_SESSION_BUS_ADDRESS`.

### CODE REVIEW

File: data/dbus/session.conf [hunk 1]
```diff
@@ -0,0 +1,126 @@
+<!-- This configuration file controls Anaconda's special session message bus.
+     It is based on the /usr/share/dbus-1/session.conf file.
+
+     Author(s): Jiri Konecny <jkonecny@redhat.com
+-->
+
```
Comment (poncovka) [inline data/dbus/session.conf]: Please, change the name of the file.

**Comment by jkonecny12** [comment]
jenkins, test this please

File: data/dbus/session.conf (hunk 1 above)
Comment (poncovka) [reply data/dbus/session.conf]: The config file looks to be too strict, but that is fine for now.

Comment (jkonecny12) [reply data/dbus/session.conf]: We can change the policy anytime we want.

**Comment by jkonecny12** [comment]
**UPDATED:**
* Rename `session.conf` to `anaconda-bus.conf`.
* Fix check if dbus exits properly.

**Pull request merged**